### ISSUE TITLE
Add amenity=give_box

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/iD#7574]
How about adding a recently approved preset?
It hasn't been used many times, but having its preset in iD, people could discover this tag.
https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dgive_box

**Comment by quincylvania** [comment]
Neat tag. I used the alternative name "Free Box" for the English locale since I'm certain people would mistag donation-only drop-off boxes as "give boxes".

**Comment by forteller** [comment]
I still can't find this on https://preview.ideditor.com/master/ ?

**Comment by quincylvania** [comment]
@forteller That v3 preview site isn't getting updated anymore. You can find the latest working changes at https://ideditor.netlify.app.

**Comment by forteller** [comment]
Huh. Ok, thanks. So there's no way to test the latest v3 any more? Does that mean it's close to release?

**Comment by quincylvania** [comment]
@forteller That preview site is up to date with the v3 branch, but we're not working on that branch of code anymore. The v3 preview ended up being a prototype where we tried a bunch of things that weren't fully baked. It was turning into a massive amount of work to keep that branch up-to-date with all the release-ready stuff in the `develop` branch, so we decided to abandon it.

We've learned a lot from it and will try to bring the best ideas to a future version of iD if we have the resources and community desire to do so. #7496 is tracking some of this but probably won't be merged in its current form.
